src/renderer/components/helpers/ChatMethods.ts

```diff
@@ -217,7 +217,7 @@ export async function createChatByContactIdAndSelectIt(
   )
 
   if (!chatId) {
-    throw new Error(window.static_translate('create_chat_error_desktop'))
+    throw new Error('could not create chat')
```

Comment:
This error is in one of 3 code paths shown to the user, But anyways should not happen and is not useful, so no need to translate it (`createChatByContactId` would normally throw a more useful error before this if is reached I think).